custodial-copy-backend/src/main/scala/uk/gov/nationalarchives/custodialcopy/Processor.scala

```diff
@@ -324,6 +324,9 @@ class Processor(
       _ <- ocflService.createObjects(changedObjectsPaths)
       _ <- logger.info(s"${changedObjectsPaths.length} objects updated")
 
+      _ <- missingObjectsPaths.parTraverse(missingObjectPath => Files[IO].deleteIfExists(Path(missingObjectPath.sourceNioFilePath.toString)))
```

Comment:
Could you do `Path.fromNioPath(missingObjectPath.sourceNioFilePath)` and `Path.fromNioPath(changedObjectPath.sourceNioFilePath)` please. Feels a bit cleaner than calling `toString`